src/app/collection/pages/collection-ncyte/feature/collection-learning-object-card/collection-learning-object-card.component.html

```diff
@@ -0,0 +1,31 @@
+<div class =  "container">
+    <div> 
+    <div *ngIf = "learnObj.length !== 'micromodule'">
+        <div class="module-background">
+            <img class ="background"src="../../../../../../../assets/images/michael-dziedzic-nLFqr9Mr9H8-unsplash 1.png" alt="image not shown">
+            <img class = "module" src="../../../../../../../assets/images/MODULE.png" alt="image not shown">
+        </div>
+    </div>
+    <div *ngIf = "learnObj.length === 'micromodule'">
+        <div class= "module-background" >
+            <img class = "background"src="../../../../../../../assets/images/adi-goldstein-EUsVwEOsblE-unsplash 2.png" alt="">
+            <img class = "micromodule" src="../../../../../../../assets/images/MICROMODULE.png" alt="image not shown">
```

Comment:
You don't need an image for the micromodule. You can have a div above the picture that has the word micromodule as its content.